server/modifiers.py

```diff
@@ -203,3 +204,98 @@ def insert_empty_statements(self) -> Self:
         self.modified_count += 1
 
         return self
+
+    def reverse_booleans(self) -> Self:
+        """A code modifier that messes up some conditions.
+
+        This will reverse True and False keywords.
+
+        Returns:
+            The modifier instance.
+        """
+        number_boolean_pairs = []
+        for num, line in enumerate(self.file_contents):
+            if any(key in line for key in ["True", "False"]):
+                number_boolean_pairs.extend([(num, key) for key in ["True", "False"] if key in line])
+
+        line_subset = random.sample(number_boolean_pairs, min(self.difficulty, len(number_boolean_pairs)))
+        for num, key in line_subset:
+            self.modified_contents[num] = self.modified_contents[num].replace(key, str(bool(bool(key)-1)))
+            self.modified_count += 1
+
+        return self
+
+    def break_equals_statement(self) -> Self:
+        """A code modifier that causes a SyntaxError.
+
+        This will change == to =.
+
+        Returns:
+            The modifier instance.
+        """
+        line_numbers = []
+        for num, line in enumerate(self.file_contents):
+            if "==" in line:
+                line_numbers.append(num)
+
+        line_subset = random.sample(line_numbers, min(self.difficulty, len(line_numbers)))
+        for num in line_subset:
+            self.modified_contents[num] = self.modified_contents[num].replace("==", "=")
+            self.modified_count += 1
+
+        return self
+
+    def mix_type_keywords(self) -> Self:
+        """A code modifier that causes a ValueError.
+
+        This will mix type keywords.
+
+        Returns:
+            The modifier instance.
+        """
+        number_type_pairs = []
+        for num, line in enumerate(self.file_contents):
+            if any(key in line for key in TYPES):
```

Comment:
This also catches type hints, for example `def say_hello() -> str:\n`. Is this intended or for this function, do you mean using the types to cast values such as `int("5")`?

Reply:
Well at first it was not intended but then i thought about it as a feature

Reply:
I think leaving it in will add to the theme of 'It's a feature, not a bug'. Perhaps give a small description of this in the documentation